ext/lib/ipc/open-amp/open-amp/docs/img-src/gen-graph.py

```diff
@@ -1,4 +1,3 @@
-from graphviz import Digraph
```

Comment:
This patches file in ext/.

Reply:
Is that strictly verboten? A bunch of other `ext/` changes got in earlier. :)

Reply:
> Is that strictly verboten? A bunch of other ext/ changes got in earlier. :)

No, just FYI. I believe we started with a requirement "if ext/ is patched, the patch should be submitted upstream and Zephyr's commit message should include reference to upstream patch", but I'd say following that lies largely on patch author's side. But I hope you'd agree that it's easy to get into situation when one guy patches something in ext/, then another guy pushes new upstream release and overwrite any changes back.

Reply:
Made an upstream PR: https://github.com/OpenAMP/open-amp/pull/168